code/modules/mob/living/carbon/human/species_types/ethereal.dm

```diff
@@ -69,6 +69,7 @@
 	RegisterSignal(ethereal, COMSIG_ATOM_EMAG_ACT, PROC_REF(on_emag_act))
 	RegisterSignal(ethereal, COMSIG_ATOM_EMP_ACT, PROC_REF(on_emp_act))
 	RegisterSignal(ethereal, COMSIG_LIGHT_EATER_ACT, PROC_REF(on_light_eater))
+	RegisterSignal(ethereal, COMSIG_HIT_BY_SABOTEUR, PROC_REF(on_saboteur))
```

Comment:
Are ethereals readded elsewhere in the code or do they not exist at all?

Reply:
readded somewhere else, i should prolly fix that